api/python/provisioner/commands/validator/validator.py

```diff
@@ -255,3 +295,210 @@ class YumRepoDataValidator(DirValidator):
     def __init__(self):
         super().__init__({Path("repomd.xml"): FileValidator(required=True)},
                          required=True)
+
+
+@attr.s(auto_attribs=True)
+class FileContentScheme(ABC):
```

Comment:
(optional) I guess ABC is not needed for the moment

Reply:
Improved.